app/assets/javascripts/event_staff.js

```diff
@@ -17,7 +32,47 @@ $(document).ready(function(){
     }
   });
 
+<<<<<<< HEAD
   $(".event_staffs").on("click", ".remove_staff", function(){
     $(this).parents(".chip").hide("slow", function() { $(this).remove(); });
   });
 });
+=======
+    $(".add_staff_field").autocomplete({
+        delay:500,
+        minLength: 4,
+        source: "/lookup_staffs",
+        select:function(event, ui){
+          var staffId = ui.item.data;
+          $("#staff_id").val(staffId);
+        }
+    });
+
+
+      $(".event_staffs").on("click", ".remove_staff", function(){
+        $(this).parents(".chip").hide("slow", function() { $(this).remove(); });
+      });
+
+      $("#add_staff").on("click", function(){
+          var staffId = $("#staff_id").val(),
+              email = $("#staff_field_id"),
+              staffRole;
+
+          if (validateEmailField(email, staffId)) {
+            return;
+          }
+
+          staffRole = $("[name='role']").val();
+          $.get("/user_info/" +staffId +"?role=" + staffRole, function(data){
+            if (isExistingOnPage(data.user_id)){
+              notify("This user has been added already");
+            }else{
+              $("#event_staffs").append(generateHtml(data));
+              notify("Staff has been successfuly added");
+              $("#staff_field_id").val("");
+              $("#staff_id").val("");
+            }
+        });
+      });
+    });
+>>>>>>> 5d4aa35... jcbajkbcka
```

Comment:
Unexpected use of '>>>'.<br>Expected an identifier and instead saw '>>>'.<br>Unexpected '5'.<br>Unexpected early end of program.